### DISCUSSION TITLE
permissions is undefined in SETTINGS > COMMENTS PLUGIN > CONFIGURATION (strapi latest)

### DISCUSSION THREAD

**Discussion by MarinosTBH** [VirtusLab-Open-Source/strapi-plugin-comments#244] [Q&A]
I just followed this tuto line by line : https://strapi.io/blog/how-to-build-a-photo-sharing-app-with-nuxt-3-graph-ql-cloudinary-postgres-and-strapi
when i access => SETTINGS > COMMENTS PLUGIN > CONFIGURATION
i get this error :TypeError permissions is undefined,
even though i followed the guide as it is ?

PS: Before that i tried to install Comments in my own project and it show the same exact problem.

Details in Attachments: 
![image](https://github.com/VirtusLab-Open-Source/strapi-plugin-comments/assets/96175731/fb119362-211b-4c77-89df-83671c9c0e7e)
![image](https://github.com/VirtusLab-Open-Source/strapi-plugin-comments/assets/96175731/3add4077-6591-4997-bcb1-98d2e96595b5)

**Comment by cyp3rius** [comment]
@MarinosTBH this should be reported as issue and seems last version got fix. Please check.
